internal/jobs/launch_instance_aws.go

```diff
@@ -308,20 +308,27 @@ func FetchInstancesDescriptionAWS(ctx context.Context, args *LaunchInstanceAWSTa
 			span.SetStatus(codes.Error, "cannot get list instances description")
 			return fmt.Errorf("cannot get list instances description: %w", errRetry)
 		}
+		logger.Trace().Msgf("AWS returned %d instance details", len(instancesDescriptionList))
 
 		if len(instancesDescriptionList) == 0 {
 			return ErrTryAgain
 		}
 
-		for _, instance := range instancesDescriptionList {
+		for i, instance := range instancesDescriptionList {
+			logger.Trace().Msgf("Instance id %s private IPv4:%s public IPv4:%s DNS:%s",
+				instancesIDList[i],
+				instance.PrivateIPv4,
+				instance.IPv4,
+				instance.DNS,
+			)
 			errRetry := rDao.UpdateReservationInstance(ctx, args.ReservationID, instance)
 			if errRetry != nil {
 				span.SetStatus(codes.Error, "cannot update instance description")
 				return fmt.Errorf("cannot update instance description: %w", errRetry)
 			}
 		}
 		return nil
-	}, 1000, 500, 500, 1000, 2000)
+	}, 500, 500, 1000, 1000, 2000)
```

Comment:
I tested numbers as short as 10ms and AWS is blazing fast, it looks like it has the IP addresses prepared way before we even attempt to get. So shortening from 1 second initial lookup to just 1/2 second. Also added some logging, it was useful for debugging.